lua/plenary/path.lua

```diff
@@ -32,6 +32,13 @@ path.sep = (function()
   end
 end)()
 
+path.root = function(base)
+  -- Handle Unix-like paths. The root is always '/'
+  if path.sep == '/' then return '/' end
```

Comment:
We should do what you had before in terms of setup. So the os check only happens once. First time the script runs and sets the function.

```lua
path.root = (function()
  if linux then
    return function() return '/' end
  else
    return ...
  end
end)()
```

Reply:
Ah I see. It's implemented now at least 😃

Reply:
Once more indention :) other LGTM thanks

Reply:
Dangit! It's fixed now